tecs/tecs.cpp

```diff
@@ -315,8 +314,8 @@ void TECS::_update_throttle_setpoint(const float throttle_cruise, const matrix::
 
 	// Calculate the throttle demand
 	if (_underspeed_detected) {
-		// always use full throttle to recover from an underspeed condition
-		_throttle_setpoint = 1.0f;
+		// always use max throttle to recover from an underspeed condition
+		_throttle_setpoint = _throttle_setpoint_max;
```

Comment:
@almaaro maybe we should break this part off and merge immediately?

Reply:
Sure. However, I'd like to keep this PR still open in order to address the throttle requirement for an underspeed condition (that currently the thr has to be over 95% of the max value regardless of how low the airspeed is).